scripts/language_model/word_language_model.py

```diff
@@ -178,25 +177,114 @@
                       'epsilon': 1e-9}
 
 trainer = gluon.Trainer(model.collect_params(), args.optimizer, trainer_params)
+
 loss = gluon.loss.SoftmaxCrossEntropyLoss()
+ar_loss = nlp.loss.ActivationRegularizationLoss(args.alpha)
+tar_loss = nlp.loss.TemporalActivationRegularizationLoss(args.beta)
+
+
+class JointActivationRegularizationLoss(gluon.loss.Loss):
+    r"""Computes Joint Regularization Loss with standard loss.
+
+    The activation regularization refer to
+    gluonnlp.loss.ActivationRegularizationLoss.
+
+    The temporal activation regularization refer to
+    gluonnlp.loss.TemporalActivationRegularizationLoss.
+
+    Parameters
+    ----------
+    loss : gluon.loss.Loss
+        The standard loss
+    ar_loss: gluonnlp.loss.ActivationRegularizationLoss
+        The activation regularization
+    tar_loss: gluonnlp.loss.TemporalActivationRegularizationLoss
+        The temporal activation regularization
+
+    Inputs:
+        - **out**: NDArray
+        output tensor with shape `(sequence_length, batch_size, input_size)`
+          when `layout` is "TNC".
+        - **target**: NDArray
+        target tensor with shape `(sequence_length, batch_size, input_size)`
+          when `layout` is "TNC".
+        - **states**: the stack outputs from RNN,
+        which consists of output from each time step (TNC).
+        - **dropped_states**: the stack outputs from RNN with dropout,
+        which consists of output from each time step (TNC).
+
+    Outputs:
+        - **loss**: loss tensor with shape (batch_size,). Dimensions other than
+          batch_axis are averaged out.
+    """
+
+    def __init__(self, l, ar_l, tar_l, weight=None, batch_axis=None, **kwargs):
+        super(JointActivationRegularizationLoss, self).__init__(weight, batch_axis, **kwargs)
+        self._loss = l
+        self._ar_loss = ar_l
+        self._tar_loss = tar_l
+
+    def __repr__(self):
+        s = 'JointActivationTemporalActivationRegularizationLoss'
+        return s
+
+    def hybrid_forward(self, F, out, target, states, dropped_states): # pylint: disable=arguments-differ
+        # pylint: disable=unused-argument
+        l = self._loss(out.reshape(-3, -1), target.reshape(-1,))
+        l = l + self._ar_loss(*dropped_states)
+        l = l + self._tar_loss(*states)
+        return l
+
+
+joint_loss = JointActivationRegularizationLoss(loss, ar_loss, tar_loss)
```

Comment:
hybridize for faster speed

Reply:
The hybridize doesn't work for joint_loss since  ``mxnet.base.MXNetError: Invalid Parameter format for reverse expect boolean but value='-1', in operator Reshape(name="", reverse="-1", shape="-3")``

Reply:
This can be fixed if you use `out.reshape((-3, -1))` instead. Given that there doesn't seem to be any operator preventing hybridization in this loss, would you mind making a follow-up PR to make hybridization work for this loss?

Reply:
I think it is better to make a separate PR for this. For now, it is using unfixed sized lists, it will still have issue when using ``out.reshape((-3, -1))``

Reply:
Sure, that's OK.